File: two_sum.rb
def two_sum(nums, target)
    hash = {}

    nums.each_with_index do |num,i|
        hash[i] = num
    end

   answer = hash.collect do |key,value|
        wanted  = target-value
        if hash.has_value?(wanted) && hash.key(wanted) != key
            return array=[key,hash.key(wanted)]
            break
        end
    end

    return answer
end

File: two_sum.py
 def twoSum(self, nums: List[int], target: int) -> List[int]:

        array = []
        for  idx, num in enumerate(nums):
            for  idx2, num2 in enumerate(nums):
                if num + num2 == target and idx != idx2:
                    return([idx,idx2])

                    
